client/src/redux/action-creators/index.js

```diff
@@ -3,13 +3,16 @@ import { API_KEY, API_URL } from '../../config/config.js';
 
 //ACTION CREATORS
 
-export const fetchQuestionList = () => {
+export const fetchQuestionList = (product_id, page, count) => {
   return (dispatch) => {
     axios.get(`${API_URL}/qa/questions`,
     {
       headers: {Authorization: API_KEY},
       params: {
-      product_id: 16056
+      // product_id: product_id,
+      product_id: 16056,
```

Comment:
This might be a good place for a future update to add the productId from the store state so the question list updates dynamically

Reply:
Will do :)